en/docs/deploy-and-publish/deploy-on-gateway/api-microgateway/deploy-api/deploy-api-to-microgateway.md

```diff
@@ -0,0 +1,146 @@
+
+### Step 1 - Download apictl and Set the Path Variable 
+
+First download [apictl](https://github.com/wso2/product-apim-tooling/releases) locally and extract it into a folder of your choice. Then, add its location to your PATH variable.
+
+```
+export PATH=$PATH:<directory-of-the-apictl-executable>
+```
+
+### Step 2 - Create an API Project using apictl
+
+Let's create your first project "petstore" using an open API definition. The following `apictl init` command will generate a project folder containing all the necessary files.
+
+!!! warning
+
+    If you have used a previous version of apictl before, remember to delete the directories `.wso2apictl` and `.wso2apictl.local` that are located in your home directory. Deleting them will make the newer apictl create them again, with content compatible with the current version.
+
+``` bash
+apictl init petstore --oas https://petstore.swagger.io/v2/swagger.json
+```
+
+### Step 3 - Add an Mcrogateway Environment to apictl
+
+To use apictl for Microgateway, a Microgateway environment needs to be added to apictl. This environment will hold the adapter URL for further commands.
+
+``` bash
+apictl mg add env dev --adapter https://localhost:9843
+```
+
+!!! info
+
+    Note `mg` in the above command. The apictl commands that starts as `apictl mg` are microgateway specific. If a command does not have `mg` after `apictl` then the command could probably be common to both Microgateway and API Manager, but it could also be API Manager specific. 
+
+!!! tip
+
+    The apictl commands here onwards are executed with the -k flag to avoid SSL verification with the microgateway.
+
+    To communicate via https without skipping SSL verification (without -k flag), add the cert in the Microgateway truststore into the `<USER_HOME>/.wso2apictl/certs` folder.
+
+### Step 4 - Login to the Mcrogateway Environment in apictl
+
+You can use the following command to login to the above microgateway cluster (in other words login to the microgateway adapter). By logging in, an access token will be retrived from the microgateway and saved in apictl.
+
+``` bash
+apictl mg login dev -k
+```
+
+or
+
+``` bash
+apictl mg login dev -u admin -p admin -k
+```
+
+### Step 5 - Deploy the API
+
+Now let's deploy the API to Microgateway by executing the following command.
+
+``` bash
+apictl mg deploy api -f <path_to_the_API_project_just_created>/petstore -e dev -k
+```
+
+That's it!
+
+More apictl commands related to microgateway can be found at [Managing Microgateway with apictl]({{base_path}}/install-and-setup/setup/api-controller/managing-microgateways/managing-microgateways-with-ctl)
+
+## Invoke the API
+{! ./includes/obtain-jwt.md !}
```

Comment:
Was his tested locally?

Reply:
Yes, both the cases were tested locally.